massgov/mayflower
# React/DP-12293: Currency component.

## Description
This adds the Input and InputCurrency components to mayflower react. This PR adds the [Numbro library](https://numbrojs.com/) for support currencies.

Both Input and InputCurrency are exported here, as well as Input's context, so that anybody using mayflower react could potentially use Input for their own input components.

## Related Issue / Ticket

- [JIRA issue](https://jira.mass.gov/browse/DP-12293)
- [Github issue]()

## Steps to Test
1. Pull the branch and run  `npm install`, then start up storybook. Browse to http://localhost:6006 then go to Atoms -> forms -> InputCurrency. Verify there's no errors and that all knob settings work.

## Screenshots
![screencapture-localhost-6006-2019-01-24-10_57_18](https://user-images.githubusercontent.com/5257660/51690760-56fec180-1fc7-11e9-84b2-e57e88a8199c.png)
